cime/src/drivers/mct/cime_config/namelist_definition_drv.xml

```diff
@@ -4749,4 +4749,17 @@
     </values>
   </entry>
 
+  <entry id="seq_flux_atmocn_minwind">
+    <type>real</type>
+    <category>seq_flux_mct</category>
+    <group>seq_flux_mct_inparm</group>
+    <desc>
+       minimum wind speed for atmOcn flux calculations
+    </desc>
+    <values>
+      <value>0.5</value>
+      <value aqua_planet_sst_type="sst_aquap11">1.0</value>
```

Comment:
We need to change "sst_aquap11" to "sst_aquap_constant"

Reply:
I can do this, but I'm not sure how. Should I commit directly to this branch? Or to a CIME branch?

Reply:
@whannah1 , thanks for looking at this. I will make the change.

Reply:
@whannah1 , ok, see if you approve of my change.